go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/omec-project/smf
 
-go 1.21
+go 1.23
```

Comment:
@gruyaume, the Dependabot opened a few dependency-related PRs. So, I had to update the GO version in the first of them (https://github.com/omec-project/smf/pull/352/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R3) for me to be able to test and approve those PRs. You can remove this change because the GO version is already updated in the latest source code (go.mod)

Reply:
Done